src/getHumanAge.test.js

```diff
@@ -7,4 +7,52 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return a correct dog and cats years for an old animals', () => {
+    const result = getHumanAge(100, 100);
+
+    expect(result).toEqual([21, 17]);
+  });
+
+  test('should return zero for 0 array', () => {
+    const result = getHumanAge(0, 0);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test('should return zero for animals younger 15 years', () => {
```

Comment:
The test description 'should return zero for animals younger 15 years' might be misleading. It should specify that it applies to both cats and dogs.